### PR TITLE
added Linux docset

### PR DESCRIPTION
**Pull request by Yanpas**
2nd try

### CODE REVIEW

File: docsets/Linux_Man_Pages/docset.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Linux Man Pages",
+    "version": "/1.0.0",
+    "archive": "Linux.tgz",
```
Comment (Kapeli) [inline docsets/Linux_Man_Pages/docset.json]: This needs to be changed to `Linux_Man_Pages.tgz`

Comment (Yanpas) [inline docsets/Linux_Man_Pages/docset.json]: updated

**Comment by Kapeli** [comment]
Thank you! 👍

**Comment by Yanpas** [comment]
I'll update python script according to name changes soon.

**Pull request merged**